Index: fsbl/Makefile
===================================================================
--- fsbl.orig/Makefile
+++ fsbl/Makefile
@@ -2,7 +2,7 @@
 
 CC := arm-none-eabi-gcc
 CC_FLAGS := -MMD -MP       -mcpu=cortex-a9 -mfpu=vfpv3 -mfloat-abi=hard 
-CFLAGS := 
+CFLAGS := -fdata-sections -ffunction-sections
 LN_FLAGS :=  -mcpu=cortex-a9 -mfpu=vfpv3 -mfloat-abi=hard -Wl,-build-id=none -specs=Xilinx.spec  -lrsa -Wl,--start-group,-lxil,-lgcc,-lc,--end-group -Wl,--start-group,-lxilffs,-lxil,-lgcc,-lc,--end-group -Wl,--start-group,-lrsa,-lxil,-lgcc,-lc,--end-group  -Wl,--gc-sections
 
 c_SOURCES := $(wildcard *.c)
